SPECS/iptables/ip6save

```diff
@@ -2,12 +2,12 @@
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
+:OUTPUT ACCEPT [0:0]
 # Allow local-only connections
 -A INPUT -i lo -j ACCEPT
 -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+-A INPUT -p icmpv6 -j ACCEPT
```

Comment:
Similar concerns and questions here.

Reply:
So, from an IPv6 perspective, icmp is essential for v6. For example, arp is replaced by NDP with ndp operating over icmp

Reply:
Correct, if icmpv6 is completely disabled the server cannot function properly as Router Advertisement and Neighbor Discovery is done over icmpv6

Reply:
If you want, I can reduce it to the essential icmp messages for V6. I believe that the following are essential for ipv6 operation

```
-p udp  --dport 546 -d fe80::/64 
-A -p icmpv6 ---icmpv6-type 133 -j ACCEPT
-A -p icmpv6 --icmpv6-type 134  -j ACCEPT
-A -p icmpv6 --icmpv6-type 135  -j ACCEPT
-A -p icmpv6 --icmpv6-type 136  -j ACCEPT
-A -p icmpv6 --icmpv6-type 137  -j ACCEPT
```

https://www.iana.org/assignments/icmpv6-parameters/icmpv6-parameters.xhtml

Reply:
Another option missing is type 2 Packet Too Big PTB, this is important for the host to receive it as it may have to retry the communication with a smaller packet.